src/lib/crypto/kimchi_bindings/js/js_backend/bigint_helpers.js

```diff
@@ -0,0 +1,69 @@
+/* global joo_global_object, caml_create_bytes, caml_bytes_unsafe_set, caml_bytes_unsafe_get, caml_ml_bytes_length
+*/
+
+// Provides: BigInt_
+var BigInt_ = joo_global_object.BigInt;
+// Provides: Uint8Array_
+var Uint8Array_ = joo_global_object.Uint8Array;
+// Provides: _0n
+var _0n = joo_global_object.BigInt(0);
+// Provides: _1n
+var _1n = joo_global_object.BigInt(1);
+// Provides: _2n
+var _2n = joo_global_object.BigInt(2);
+// Provides: _8n
+var _8n = joo_global_object.BigInt(8);
+// Provides: _32n
+var _32n = joo_global_object.BigInt(32);
+
+// Provides: caml_bigint_to_bytes
+// Requires: BigInt_, Uint8Array_, _8n
+function caml_bigint_to_bytes(x, length) {
```

Comment:
no dedicated test. it seems to be tested implicitly by a combination of things; if I change the encoding in both `to_bytes` and `of_bytes`, I always get errors like `Invalid_curve_point`, i.e. some hard coded byte sequence doesn't successfully deserialize. If I only change `to_bytes` but not `of_bytes`, then I get errors in a couple of client SDK / mina-signer tests, because signatures can't be verified (to_bytes is used in some to_base58_check functions)